build-logic/build-update-utils/src/main/kotlin/gradlebuild.check-update-release-note.gradle.kts

```diff
@@ -0,0 +1,9 @@
+import gradlebuild.buildutils.tasks.CheckContributorsInReleaseNote
+import gradlebuild.buildutils.tasks.UpdateContributorsInReleaseNote
+
+tasks.register<CheckContributorsInReleaseNote>("checkContributorsInReleaseNote") {
```

Comment:
It seems like this should go to the `docs` plugins. Then you also don't need to hardcode the location of the release notes here.

Reply:
I found a better place: `ReleaseNotesPlugin`. Now these tasks are in `docs` subproject.